### ISSUE TITLE
Some text strings cut-off with French localisation

### ISSUE THREAD

**Issue by PoorPocketsMcNewHold** [badanka/Compact#41]
[Found this cuted line,](https://lewd.pics/p/?i=TzQ4.png) which should probably be displayed as "articles téléchargés". I will try to search if more case is present in my native language, but these kinds of issues seem to be happening depending on the language and on their specific translation, so, some adjustments dedied to each reported language would need to be done, in order to fix this issue as done by the [Air For Steam skin](https://github.com/Outsetini/Air-for-Steam/wiki#how-to-fix-overlay-text-being-cut-off-in-other-languages).

**Comment by badanka** [comment]
Try [this one](https://github.com/badanka/Compact/blob/master/resource/layout/steamrootdialog.layout) and see if it helps.

There are probably several of these that can be found for different languages. I've done a ton of adjustments for Turkish and Polish over the years with the help of a few users. A lot of the time 'width=max' would be the preferable setting, but things often break for no apparent reason if you set that, so you need to kind of wing it.

**Comment by PoorPocketsMcNewHold** [comment]
[Great, Maybe too much as it's not directly stacked on the border](https://lewd.pics/p/?i=3qtI.png), but that fine for myself. Should I let this issue open for other people to report their inconsistencies with their language as well, if I find more issues with the French language, or close it? _(I would still open it again, for the second case personally)_

**Comment by badanka** [comment]
There's supposed to be a gap there, since you have the loading icon for loading web pages sitting in the far-right corner (the "throbber"). Just open this issue again if you find other problems with French localisation in Compact.

**Comment by PoorPocketsMcNewHold** [comment]
[Talking about other inconsistencies, Find another one.](https://lewd.pics/p/?i=7sP0.png) According to STS: The entire string of this last cut setting is 

> M'informer des ajouts ou modifications apportées à mes jeux, des nouvelles sorties et des sorties à venir.

[And also found this one.](https://lewd.pics/p/?i=VpTB.png) when clicking on the little blue triangle next to the community name.
EDIT: Didn't want to post another comment again, Since you will gonna work on this later, and didn't wanted to bother you on it again right now. Those should be the last ones.
[I have no idea if the skin actually supports the musical player](https://lewd.pics/p/?i=A7dO.png), couldn't try with real music playing, Steam didn't found any music at all despite finding the albums correctly.
It's pretty difficult to see those, but i assume that the buttons superposing with the principal buttons are : 

> Ajouter à la liste
Ajouter à la Playlist
Ajouter à la liste
Ajouter à la Playlist
Montrer l'album
Monter sur le système

So definitely, two strings in double, and the last one cut in half _(But still working)_
[And the recent player's list](https://lewd.pics/p/?i=MxnS.png), Might not really be that much of a problem, as it's displayed "properly cut" (Meaning, with the proper ...) It should be "Parties précédentes"

**Comment by badanka** [comment]
Ugh, the second one shouldn't even behave like that. I'll take a look at it later, busy with some other stuff at the moment.

**Comment by badanka** [comment]
The first one, in Settings -> Interface, has been fixed. Put [this file](https://github.com/badanka/Compact/blob/master/steam/cached/SettingsSubInterface.res) in _Steam\skins\Compact\steam\cached\_ and restart.

I can't reproduce the second one (the account menu). I changed to French on my second computer and the menu has the correct width there. Have you modified any files on your own? Maybe resource\layout\accountmenu.layout ?

I can't reproduce the third one either. You're either supposed to get a button that says "Play album" or "Play tracks". You seem to be getting both buttons at once, which I've never seen happen. Does it happen with all your albums or just that one? And again, have you modified any files on your own?

I can confirm the fourth one (View -› Players). I'm currently trying to find a way to fix it.

**Comment by PoorPocketsMcNewHold** [comment]
Working great, That's one is fixed indeed.

Outside of all the recommanded modifications i've done in this issue thread, no outside modifications has been made, and not even in the Steam executable itself (Developer console for example) But i do have the latest open beta installed and running of Steam. For this one, It seems that the currency in my wallet was the cause.
![image](https://user-images.githubusercontent.com/13170204/43356027-3988ced4-9269-11e8-8ba6-674a2859255e.png)

It's currently happening to every albums that i own, but since i'm having an issue, making the Steam Player to not found any of my music files itself, despite founding the albums properly (And updating them correctly), Until i find what it's been causing from, We should maybe ignore that one as it could be due to this bug. 
![2018-07-28_13-28-20](https://user-images.githubusercontent.com/13170204/43356077-3426ad70-926a-11e8-9785-d9c458ee098b.gif)

**Comment by PoorPocketsMcNewHold** [comment]
For the last music player bug, I was right. It was indeed due to the bug. I've been able to fix it by refreshing the founded albums completely, and I've been able to get my sounds files back playing and fix that interpolating looking issue.

**Comment by badanka** [comment]
@PoorPocketsMcNewHold Try [this file](https://github.com/badanka/Compact/blob/master/resource/styles/steam.styles) and put it in _\Steam\resource\styles_ for a fix for the truncated text in View -› Players. It seems to be a bug in the client where it doesn't automatically adjust the width of the tab to fit the text so I had to force a minimum-width for it. I set it to fit the French translation, so if someone who's using a different language that has an even longer translation string for it I guess I'll have to adjust the width further.

That should've been the last one of the ones you reported right?
